enterprise/coderd/provisionerkeys_test.go

```diff
@@ -134,3 +134,63 @@ func TestProvisionerKeys(t *testing.T) {
 	err = orgAdmin.DeleteProvisionerKey(ctx, owner.OrganizationID, codersdk.ProvisionerKeyNamePSK)
 	require.ErrorContains(t, err, "reserved")
 }
+
+func TestGetProvisionerKey(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name        string
+		useFakeKey  bool
+		fakeKey     string
+		success     bool
+		expectedErr string
+	}{
+		{
+			name:        "ok",
```

Comment:
@defelmnq there's only one test case in the table?
You mentioned you added a few tests - where are those?

Reply:
Yeah - have seen here I think something weird happened with the rebase - my bad. 

I added back all the tests cases 🙏